test/contrib/test_tfp.py

```diff
@@ -251,3 +251,45 @@ def test_sample_tfp_distributions():
     # test intermediates are []
     value, intermediates = d(sample_intermediates=True, rng_key=random.PRNGKey(0))
     assert intermediates == []
+
+
+# test that sampling from unwrapped tensorflow_probability distributions works as
+# expected using numpyro.sample primitive
+@pytest.mark.parametrize(
+    "dist,args",
+    [
+        ["Bernoulli", (0,)],
+        ["Beta", (1, 1)],
+        ["Binomial", (10, 0)],
+        ["Categorical", ([0, 1, -1],)],
+        ["Cauchy", (0, 1)],
+        ["Dirichlet", ([1, 2, 0.5],)],
+        ["Exponential", (1,)],
+        ["Normal", (0, 1)],
```

Comment:
Could you add this [one](https://forum.pyro.ai/t/sample-from-the-mixture-same-family-distribution/3178/3?u=fehiepsi) to exercise that mixture distribution?

Reply:
Had to make a separate test for this as I can't use `tfd` arguments in the `parametrize`, but this is now added.